docs/modules/ROOT/assets/images/fire_oracle_registration.md

```diff
@@ -0,0 +1,17 @@
+# Fire oracle registration
+
+```mermaid
+sequenceDiagram
+    actor OP as OracleProvider
+    actor IO as InstanceOperator
+    participant G as GIF
+    participant O as FireOracle
+ 
+    
+    IO ->> G: grant role to OracleProvider
+    OP->>+ O: deploy FireOracle contract
```

Comment:
Since deployment can happen independently before the rest of the actions, i would put this as the first interaction.